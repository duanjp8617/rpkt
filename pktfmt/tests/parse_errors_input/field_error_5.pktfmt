packet FakeProto {
    header = [
        fake_field1 = Field {
            bit = 16,
            repr = &[u8],
            arg = &[u8],
            default = [0,1,2]
        }
    ]
}

