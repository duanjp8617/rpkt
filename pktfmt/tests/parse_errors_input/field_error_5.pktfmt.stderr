ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/field_error_5.pktfmt 3:23-8:9
                          ~~~~~~~
3 |         fake_field1 = Field {
4 |             bit = 16,
5 |             repr = &[u8],
6 |             arg = &[u8],
7 |             default = [0,1,2]
8 |         }
    ^^^^^^^^^
field error 5:
invalid default [0, 1, 2], should be 2-byte array
